src/main/java/com/quirko/logic/bricks/EasyBricks.java

```diff
@@ -0,0 +1,22 @@
+package com.quirko.logic.bricks;
+
+import java.util.List;
+import java.util.ArrayList;
+
+public class EasyBricks{
+    
+    public List<Brick> brickList;
+        
+    public EasyBricks(){
+        brickList = new ArrayList<>();
+        brickList.add(new IBrick());
```

Comment:
Her bir brick type ı bir kere eklenmeli o brick listesine. Daha sonra bir accessor metodu tanımla. O metotta her bir kolay brick typeında iki defa yeni bir listeye ekleyip o listeyi dön. HardBricks ve MediumBricks siniflarini da ayni sekilde guncelle.

brickList yerine brickTypes yap.

```
private List<Brick.class> brickListTypes;


public EasyBricks() {
...
  brickListTypes.add(IBrick.class);
  brickListTypes.add(JBrick.class);
...
}

public List<Brick> getBrickTypes() {
  return brickListTypes;
}
```

Daha sonra başka bir class tanımla ve oradan her bir typetan 2 şer tane bir listeye ekle.

class BrickListConfiguration {

  public getConfiguration() {
    List list = ...

    for (Class klass : EasyBricks.getBrickTypes()) {
      list.add(klass.newInstance());
      list.add(klass.newInstance());
    }

    for (Class klass : HardBricks.getBrickTypes()) {
      list.add(klass.newInstance());
      list.add(klass.newInstance());
    }

   ...

   return list;
}